pkg/sql/sem/tree/hide_constants.go

```diff
@@ -35,6 +35,9 @@ func (ctx *FmtCtx) formatNodeOrHideConstants(n NodeFormatter) {
 		case *Placeholder:
 			// Placeholders should be printed as placeholder markers.
 			// Deliberately empty so we format as normal.
+		case *NumVal:
```

Comment:
hmm, looking through the test changes, it seems like this NumVal change didn't actually fix any of the cases. so let's actually take out the NumVal case and could you `-rewrite` the tests again? (this should also address some of the other test issues which seem to have come up with the change)